lib/exp.py

```diff
@@ -165,6 +165,9 @@ def create_job(self, experiment, parameters):
                 'experiment_uid': experiment.uid(),
                 'experiment_name': experiment.name
             },
+            'annotations': {
```

Comment:
what other types of annotations do you foresee?  Curious to know why the job parameters weren't instead made a top-level metadata key.